datatypes.go

```diff
@@ -115,6 +128,19 @@ type TriggerData struct {
 	Tags       []string `json:"__notifier_trigger_tags"`
 }
 
+func (trigger *TriggerData) TemplateDescription(data map[string]interface{}) error {
+	buffer := new(bytes.Buffer)
+
+	triggerTemplate := template.Must(template.New("moira").Parse(trigger.Desc))
```

Comment:
In this certain case template name do not mean anything even so let's name template as a `trigger-description`